src/components/HomePage/ServiceSection/index.tsx

```diff
@@ -1,31 +1,40 @@
 import React from "react";
 import Image from "next/image";
-import { FaArrowRight, FaArrowLeft } from "react-icons/fa";
 import Button from "../../common/Button";
 import ServiceHighlights from "../../common/ServiceHighlights";
+import Link from "next/link";
 
-const highlights: string[] = [
+const highlights = [
   "Coverage for weddings, parties, corporate functions, and more.",
   "Skilled photographers who know how to seize the moment.",
   "A mix of candid and posed shots for a comprehensive story.",
   "Quick turnaround for you to relive the day's highlights.",
 ];
 
-const title: string = "Events";
-
-const description: string =
+const title = "Events";
+const description =
```

Comment:
why do you store these two variables outside the component
I'd just use literal values